### PR TITLE
chore: specify version require for 2.16

### PR DESCRIPTION
**Pull request by chenrui333**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes https://github.com/kedacore/keda-docs/issues/1498
relates to https://github.com/endoflife-date/endoflife.date/pull/6197

### CODE REVIEW

File: content/docs/2.14/deploy.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ We provide a few approaches to deploy KEDA runtime in your Kubernetes clusters:
 - [Operator Hub](#operatorhub)
 - [YAML declarations](#yaml)
 
-> 💡 **NOTE:** KEDA requires Kubernetes cluster version 1.27 and higher
+> 💡 **NOTE:** KEDA requires Kubernetes cluster version v1.27 to v1.29
```
[the remark this answers is not in the document]
Comment (JorTurFer) [reply content/docs/2.14/deploy.md:11]: @zroubalik @wozniakjan

[the remark this answers is not in the document]
Comment (chenrui333) [reply content/docs/2.14/deploy.md:11]: yeah, not sure either, but this aligns with the release table though

image

[the remark this answers is not in the document]
Comment (chenrui333) [reply content/docs/2.14/deploy.md:11]: @JorTurFer any thoughts of merging the patch? thanks!

[the remark this answers is not in the document]
Comment (zroubalik) [reply content/docs/2.14/deploy.md:11]: I agree with @JorTurFer on this one, let's be explicit.

[the remark this answers is not in the document]
Comment (chenrui333) [reply content/docs/2.14/deploy.md:11]: @zroubalik I am not very clear, can you suggest the changes?

[the remark this answers is not in the document]
Comment (JorTurFer) [reply content/docs/2.14/deploy.md:11]: Yeah, just revert these changes and keep only the changed in the file `cluster.md`

[the remark this answers is not in the document]
Comment (chenrui333) [reply content/docs/2.14/deploy.md]: sounds good, just refreshed the pr, let me know if it is better now.

**Review by JorTurFer — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @JorTurFer!

**Pull request merged**